website/static/js/project.js

```diff
@@ -136,7 +136,7 @@ $(function() {
 
             e.preventDefault();
         } else if ($(e.target).find('#title').val().length > 200) {
-            $('#alert').text('The new component title cannot be more than 200 characters.');
+            $('#alert').text('The new component title cannot be more than 200 characters.'); //This alert never appears...
```

Comment:
Can you add a div with `id=alert` (and the `text-danger` css class for styling) to `modal_add_component.mako`? It must have been deleted accidentally, because right now even the alert for an empty component title won't show.